comments.php

```diff
@@ -69,15 +69,14 @@
 		<?php
 		endif; // Check for comment navigation.
 
-	endif; // Check for have_comments().
-
+		// If comments are closed and there are comments, let's leave a little note, shall we?
+		if ( ! comments_open() ) : ?>
 
-	// If comments are closed and there are comments, let's leave a little note, shall we?
-	if ( ! comments_open() && get_comments_number() && post_type_supports( get_post_type(), 'comments' ) ) : ?>
+			<p class="no-comments"><?php esc_html_e( 'Comments are closed.', '_s' ); ?></p>
+		<?php
+		endif; // Check for comments closed.
```

Comment:
The closing comment is normally just added if there is a long if statement. In this case I don't think it is needed and I am not sure if it grammatically correct.